src/generators/vcf_file_generator.py

```diff
@@ -20,20 +20,14 @@ class VcfFileGenerator:
     file_header = """##contig=<ID=,length=,assembly={assembly},md5=,species="{species}",taxonomy=x>
 ##fileDate={datetime}
 ##fileformat=VCFv4.2
-##FILTER=<ID=q10,Description="Quality below 10">
-##FILTER=<ID=s50,Description="Less than 50% of samples have data">
-##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth">
-##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
-##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
-##FORMAT=<ID=HQ,Number=2,Type=Integer,Description="Haplotype Quality">
 ##INFO=<ID=allele_of_genes,Type=String,Number=0,Description="The genes that the Allele is located on">
-##INFO=<ID=alleles,Type=String,Number=0,Description="The alleles of the variant">
-##INFO=<ID=DP,Number=0,Type=Integer,Description="The label to be used for visual purposes">
-##INFO=<ID=hgvs_nomenclature,Type=String,Number=0,Description="the HGVS name of the allele">
-##INFO=<ID=symbol,Type=String,Number=0,Description="The human readable name of the allele">
+##INFO=<ID=alleles,Type=String,Description="The alleles of the variant">
+##INFO=<ID=hgvs_nomenclature,Type=String,Description="the HGVS name of the allele">
+##INFO=<ID=symbol,Type=String,Description="The human readable name of the allele">
+##INFO=<ID=geneLevelConsequence,Type=String,Description="VEP consequence of the variant">
+##INFO=<ID=symbol_text,Type=String,Description="Another human readable representation of the allele">
 ##phasing=partial
-##reference=
-##source=agr_file_generator"""
+##source=AGR VCF File generator"""
```

Comment:
I am not sure if you are or not but you should make sure that the header is sorted according to the command line sort command.

Reply:
I will work on that.